src/modules/launcher/Plugins/Microsoft.Plugin.Calculator/CalculateEngine.cs

```diff
@@ -1,16 +1,24 @@
-﻿// Copyright (c) Microsoft Corporation
+// Copyright (c) Microsoft Corporation
 // The Microsoft Corporation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Collections.Generic;
 using System.Globalization;
 using Mages.Core;
 
 namespace Microsoft.Plugin.Calculator
 {
     public class CalculateEngine
     {
-        private readonly Engine _magesEngine = new Engine();
+        private readonly Engine _magesEngine = new Engine(new Configuration
+        {
+            Scope = new Dictionary<string, object>
+            {
+                { "e", Math.E }, // e is not contained in the default mages engine
+            },
+        });
+        
```

Comment:
This line has whitespace but it should be emtpy.

Sorry for being so annoying, but our goal is to have as few warnings in CI as possible and also, Visual Studio treats these warnings as errors in local builds.

Reply:
All good, thanks for pointing it out!